core/src/main/java/greencity/controller/ManagementOrderController.java

```diff
@@ -371,29 +344,6 @@ public ResponseEntity<ReadAddressByOrderDto> getAddressByOrderId(
             .body(ubsManagementService.getAddressByOrderId(id));
     }
 
-    /**
-     * Controller for update order address.
-     *
-     * @return {@link OrderAddressDtoResponse}.
-     * @author Orest Mahdziak
-     */
-    @ApiOperation(value = "Update order address")
-    @ApiResponses(value = {
-        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = OrderAddressDtoResponse.class),
-        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
-        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
-        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
-    })
-    @PreAuthorize("@preAuthorizer.hasAuthority('EDIT_ORDER', authentication)")
-    @PutMapping("/update-address")
```

Comment:
What if client added wrong order addess
How will we update it?
Do we have another enpoint for it?

Reply:
yes we have another endpoint with putch method and url /update-order-page-admin-info/{id}